### ISSUE TITLE
Add vehicle.setHandling function

### ISSUE THREAD

**Issue by Matspyder51** [altmp/altv#158]
**Is your feature request related to a problem? Please describe.**
We can't actually change vehicle handling in runtime, that can be used to change vehicles damages modifiers, number of gears ... lot of possibilities

**Describe the solution you'd like**
Add a function than will change some handling datas only for the current vehicle : 
`vehicle.setHandling(key:string, val:any):void`
`vehicle.getHandling(key:string):any`

**Describe alternatives you've considered**
I don't see any other solutions

**Comment by rsxdalv** [comment]
Here's basic documentation for this feature:
https://forums.gta5-mods.com/topic/3842/tutorial-handling-meta
Core is handling data, subhandlingdata is secondary.
Implementation - changing memory at cars handling data, with key enforcing the location & type.
Minimum would be - 
```ts
setHandling(modelName: string | number, offset: 0...77, value: Buffer) // Buffer size - 4 bytes.
```

**Comment by emcifuntik** [comment]
Implemented in build #587
